AssignmentManager/LmsAssignment.cs

```diff
@@ -2,10 +2,23 @@
 
 namespace AssignmentsManager
 {
-    class LmsAssignment : TableEntity
+    internal class LmsAssignment : Assignment
```

Comment:
Why to inerrant from Assignment?

Reply:
So I won't put the properties again, but yes I just paid close attention, no need for it